Fails to compile if hyprlang is installed to unusual prefix

Installing `hyprlang` to a custom, unusual, prefix, such as `$HOME/.local` makes this fail to compile due `hyprlang.hpp` header not found. That's due a missing `PkgConfig::`  qualifier in an `src/style/impl/CMakeLists.txt` `target_link_libraries(` call.

```ls ~/.local/include
$ ls ~/.local/include -l
total 524
drwxr-xr-x  8 cn cn   4096 Mar  3 02:11 aquamarine
drwxr-xr-x 24 cn cn   4096 Mar  3 02:27 glaze
drwxr-xr-x  2 cn cn   4096 Mar  3 02:14 hyprcursor
-rw-r--r--  1 cn cn   6121 Mar  3 02:12 hyprcursor.hpp
drwxr-xr-x  5 cn cn   4096 Mar  3 02:16 hyprgraphics
drwxrwxr-x  4 cn cn   4096 Mar  3 02:27 hyprland
-rw-r--r--  1 cn cn  16098 Mar  3 02:03 hyprlang.hpp
drwxr-xr-x 10 cn cn   4096 Mar  3 02:02 hyprutils
...
```

```
$ cmake -DINSTALL_QML_PREFIX="lib/hypr/qml" -DCMAKE_PREFIX_PATH="$HOME/.local/" -B build                                                 
-- The CXX compiler identification is GNU 14.2.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- QML install dir: /home/cn/.local/lib/hypr/qml
-- Configuring done (1.6s)
-- Generating done (0.3s)
-- Build files have been written to: /home/cn/Dev/2_Tools/hypr/hyprland-qt-support/build

$ cmake --build build -j4                                                                
[  1%] Built target hyprland-quick-style_copy_qml
[  3%] Built target hyprland-quick-style_copy_res
...
[ 25%] Building CXX object src/style/impl/CMakeFiles/hyprland-quick-style-impl.dir/.rcc/qmlcache/hyprland-quick-style-impl_qmlcache_loader.cpp.o
[ 26%] Building CXX object src/style/impl/CMakeFiles/hyprland-quick-style-impl.dir/checkdelegate.cpp.o
[ 28%] Building CXX object src/style/impl/CMakeFiles/hyprland-quick-style-impl.dir/hyprland-quick-style-impl_autogen/mocs_compilation.cpp.o
[ 30%] Building CXX object src/style/impl/CMakeFiles/hyprland-quick-style-impl.dir/hyprlandstyle.cpp.o
/home/cn/Dev/2_Tools/hypr/hyprland-qt-support/src/style/impl/hyprlandstyle.cpp:2:10: fatal error: hyprlang.hpp: No such file or directory
    2 | #include <hyprlang.hpp>
      |          ^~~~~~~~~~~~~~
compilation terminated.
gmake[2]: *** [src/style/impl/CMakeFiles/hyprland-quick-style-impl.dir/build.make:174: src/style/impl/CMakeFiles/hyprland-quick-style-impl.dir/hyprlandstyle.cpp.o] Error 1
gmake[2]: *** Waiting for unfinished jobs....
gmake[1]: *** [CMakeFiles/Makefile2:674: src/style/impl/CMakeFiles/hyprland-quick-style-impl.dir/all] Error 2
gmake: *** [Makefile:136: all] Error 2
```

--- a/src/style/impl/CMakeLists.txt
+++ b/src/style/impl/CMakeLists.txt
@@ -13,6 +13,6 @@ qt_add_qml_module(hyprland-quick-style-impl
         MotionBehavior.qml
 )
 
-target_link_libraries(hyprland-quick-style-impl PRIVATE Qt::Quick hyprlang)
+target_link_libraries(hyprland-quick-style-impl PRIVATE Qt::Quick PkgConfig::hyprlang)
 
 install_qml_module(hyprland-quick-style-impl)

